docs/source/examples/managed-jobs.rst

```diff
@@ -5,43 +5,33 @@ Managed Jobs
 
 .. tip::
 
-  This feature is great for scaling out: running a single job for long durations, or running many jobs (pipelines).
+  This feature is great for scaling out: running a single job for long durations, or running many jobs in parallel.
 
-SkyPilot supports **managed jobs** (:code:`sky jobs`), which can automatically recover from any spot preemptions or hardware failures.
-It can be used in three modes:
+SkyPilot supports **managed jobs** (:code:`sky jobs`), where "managed" means
+any spot preemptions or hardware failures are auto-recovered by SkyPilot.
+Users can launch:
 
-#. :ref:`Managed Spot Jobs <spot-jobs>`: Jobs run on auto-recovering spot instances. This can **save significant costs** (e.g., up to 70\% for GPU VMs) by making preemptible spot instances useful for long-running jobs.
-#. :ref:`On-demand <on-demand>`: Jobs run on auto-recovering on-demand instances. This is useful for jobs that require guaranteed resources.
-#. :ref:`Pipelines <pipeline>`: Run pipelines that contain multiple tasks (which can have different resource requirements and ``setup``/``run`` commands). This is useful for running a sequence of tasks that depend on each other, e.g., data processing, training a model, and then running inference on it.
+.. It can be used in three modes:
+
+#. :ref:`Managed spot jobs <spot-jobs>`: Jobs run on auto-recovering spot instances. This **saves significant costs** (e.g., ~70\% for GPU VMs) by making preemptible spot instances useful for long-running jobs.
```

Comment:
For these 3 bullet points, why only the first one has "managed" in it? These are 3 different modes for managed jobs so I think we don't need to have the word "managed" in it again

Reply:
How about we rephrase them into:
1. Jobs on Spot VMs
2. Jobs on On-demand VMs
3. Pipelines

Reply:
Taken on face value
```
Jobs on Spot VMs
Jobs on On-demand/Reserved VMs
Pipelines
```
may be a bit misleading, because it mentions VMs and `sky launch` can also achieve these purposes.

Current version
```
Managed spot jobs
On-demand or reserved jobs
Pipelines
```
seems slightly better because they focus on talking about "jobs" and don't mention VMs.

Alternatively,
```
Managed spot jobs
Managed on-demand/reserved jobs
Managed pipelines
```
seems ok too, but the second item is a bit too long.

cc @Michaelvll

Reply:
How about just keep it simple since you are describing 3 *modes* not 3 entities:
```
Spot
On-demand / Reserved
Pipeline
```

Reply:
I'm mainly trying to avoid using "spot" or "on-demand" to describe jobs, as they are VM properties